bylaws.md

```diff
@@ -1,5 +1,197 @@
+
+Notwithstanding the foregoing, any of the following decisions, other than by unanimous approval by those entitled to vote, shall be valid only if the general nature of the proposal was stated in the notice of meeting or in any written waiver of notice:
+
+## Adjourned Meetings
+
+When a members’ meeting is adjourned to another time or place, notice need not be given of the adjourned meeting if the time and place thereof (or the means of electronic transmission by and to the cooperative or electronic video screen communication, if any, by which members may participate) are announced at the meeting at which the adjournment is taken.  At the adjourned meeting the cooperative may transact any business which might have been transacted at the original meeting.  If the adjournment is for more than 45 days or if after the adjournment a new record date is fixed for the adjourned meeting, a notice of the adjourned meeting shall be given to each member of record entitled to vote at the meeting.
+
+## Quorum
+
+A majority of the members shall constitute a quorum at a meeting of members.  If a quorum is present, the affirmative vote of the majority of the members entitled to vote shall be the act of the members unless the vote of a greater number is required by the articles or bylaws.
+
+The members present at a duly called or held meeting at which a quorum is present may continue to transact business until adjournment notwithstanding the withdrawal of enough members to leave less than a quorum, if any action taken (other than adjournment) is approved by at least a majority of the members required to constitute a quorum or, if required by the articles or the bylaws, the vote of the greater number so required.
+
+In the absence of a quorum, any meeting of members may be adjourned from time to time by the vote of a majority of the votes represented in person, but no other business may be transacted, except as provided in the preceding paragraph.
+
+
+## Notice of Board Meetings
+
+Regular meetings will be held the first Thursday of every month at 11:00 a.m. at the principal office of the cooperative.  If the day fixed for the regular meeting falls on a legal holiday, the meeting shall be held at the same time and place on the next business day.
+Special meetings of the board shall be held upon four days’ notice by first-class mail or 48 hours’ notice delivered personally, by telephone, including a voice messaging system, or by electronic transmission by the cooperative.  A notice, or waiver of notice, need not specify the purpose of any regular or special meeting of the board.
+
+
+
```

Comment:
```suggestion
```

**[Suggestion]:** Delete lines 196 and 197?

Reply:
updated